### DISCUSSION TITLE
Datawarehousing

### DISCUSSION THREAD

**Discussion by JorgePulp** [cynkra/dm#1801] [Show and tell]
Large databases integration with shell scripts

**Comment by JorgePulp** [comment]
I usually make shell scripts to connect, extracts, transform and load data for data models.

**Reply by krlmlr** [reply]
Thanks! Would you like to share details? Do these shell scripts make use of R or the dm package?

**Reply by JorgePulp** [reply]
Hi, yes I am talking about to charge a dimensional data model using .sh or .ksh files triggered from unix/linux scheduler (tipically ctrl-m). 
Of course variations can occur as many as diferents mainframes architectures are.
